pkg/tests/ossm/operator/clusterwide_mode_test.go

```diff
@@ -70,6 +70,29 @@ func TestClusterWideMode(t *testing.T) {
 					"expected SMMR to show 50 member namespaces, but that wasn't the case"))
 		})
 
+		t.NewSubTest("RoleBindings verification").Run(func(t test.TestHelper) {
+			t.Log("Related to OSSM-3468")
+			t.LogStep("Check that Rolebindings are not created in the member namespaces")
+			retry.UntilSuccess(t, func(t test.TestHelper) {
+				oc.Get(t, "member-0", "rolebindings", "",
+					assert.OutputContains("system:deployers",
+						"The Rolebings contains system:deployers role",
+						"The Rolebings does not contains system:deployers role"),
+					assert.OutputContains("system:image-builders",
+						"The Rolebings contains system:image-builders role",
+						"The Rolebings does not contains system:image-builders role"),
+					assert.OutputContains("system:image-pullers",
+						"The Rolebings contains system:image-pullers role",
+						"The Rolebings does not contains system:image-pullers role"),
+					assert.OutputDoesNotContain("istiod-clusterrole-basic-istio-system",
+						"The Rolebings does not contains istiod-clusterrole-basic-istio-system role",
+						"The Rolebings contains istiod-clusterrole-basic-istio-system role"),
+					assert.OutputDoesNotContain("istiod-gateway-controller-basic-istio-system",
+						"The Rolebings does not contains istiod-gateway-controller-basic-istio-system role",
+						"The Rolebings contains istiod-gateway-controller-basic-istio-system role"))
+			})
+		})
+
 		t.NewSubTest("customize SMMR").Run(func(t test.TestHelper) {
```

Comment:
Shouldn't this be multiple subtests, since many different things are tested here? It's also hard to follow what this subtest does, because it's so long.

Reply:
The only thing to not break in more sub test cases at the beginning is the dependency between the steps, but sure I can split in more subtest cases